sdk/ml/azure-ai-ml/azure/ai/ml/constants/_component.py

```diff
@@ -62,6 +62,22 @@ class DataTransferBuiltinComponentUri(object):
     EXPORT_DATABASE = "azureml://registries/azureml/components/export_data_database/versions/0.0.1"
 
 
+class LLMRAGComponentUri(object):
```

Comment:
```suggestion
class LLMRAGComponentUri:
```

Reply:
removed, but seems other objects in this file are still subclassing object